charts/lnd/templates/wallet-init-hook.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   annotations:
     "helm.sh/hook": "post-install"
 spec:
+  # FIXME: should not run as root
   securityContext:
     runAsUser: 0
```

Comment:
we could run it as non root with the new image I think?

Reply:
I tried it, didn't work. This hook actually isn't related to the lnd image at all, it just runs a bash script. The `expect` package needs to be installed, and for that we need to run this as a root user.